node/service/src/lib.rs

```diff
@@ -174,13 +174,14 @@ where
 		task_manager.spawn_essential_handle(),
 		client.clone(),
 	);
-
 	let (grandpa_block_import, grandpa_link) = sc_consensus_grandpa::block_import(
 		client.clone(),
+		GRANDPA_JUSTIFICATION_PERIOD,
```

Comment:
This change is introduced by substrate [PR #14423](https://github.com/paritytech/substrate/pull/14423)